packages/core-mobile/app/navigation/wallet/EarnScreenStack/EarnScreenStack.tsx

```diff
@@ -35,64 +36,66 @@ const EarnStack = createStackNavigator<EarnStackParamList>()
 
 function EarnScreenStack(): JSX.Element {
   return (
```

Comment:
<img width="936" alt="Screenshot 2024-08-26 at 5 27 46 PM" src="https://github.com/user-attachments/assets/a8045d1e-2a56-4c48-a6fe-7c9c3adbe2f3">

Reply:
here are a few screenshots from the current core mobile and also the slack app for references 
<img width="1000" alt="Screenshot 2024-08-26 at 5 30 44 PM" src="https://github.com/user-attachments/assets/21f24d4d-7280-43e0-bae3-3645f1beabd0">

Reply:
I fixed a lot of issues. I still need to to reproduce the time and icons missing. So I'm going to do another pass tomorrow.

Reply:
I'm unable to reproduce the status bar bug and I've tried different simulators and ios versions. I don't think I would know where the bug is coming from to try to fix it blindly. Is it still doing it on your side?

Reply:
yes it's still happening. my iphone is 13 pro max and version 16.1.2

Reply:
also can you increase the spacing to match the slack app for these 2 things?
- bottom tab icon + text and the end of screen
- bottom tab bar and the floating acting button

Reply:
and I also got the same issue about the testnet banner as @ruijialin-avalabs did. 
<img src="https://github.com/user-attachments/assets/8ba2ed6d-62b5-41fd-8585-70df6e2a58a0" width=300/>

Reply:
btw I found out how to reproduce the status bar issue. it seems to happen with:
- iOS version < 17 
- dark mode is turned off

you can download iOS version 16.1 or 16.2 in Xcode to test. here is a video:
https://github.com/user-attachments/assets/a0df9021-b7c9-401c-ab34-d5fddd79d37b

Reply:
Finally was able to reproduce and fix.